modules/comment/tests/comments.js

```diff
@@ -23,7 +23,8 @@ describe('Comments', () => {
 
   let commentId = null;
 
-  before(async () => { // Before all tests we empty the database once
+  before(async function () { // Before all tests we empty the database once
+    this.timeout(10000);
```

Comment:
We wait for the actions to complete. If the async function resolves the next action is carried out. By default the promise is expired after 2 seconds. This might not be enough in this case which is why the timeout is increased to 10 seconds.

Reply:
Aha, it configures the timeout of this test. For some reason, I have read it as a delay at the beginning of text execution.